test/clj_time/coerce_test.clj

```diff
@@ -144,6 +145,25 @@
   (is (= "1998-04-25T00:00:00.000Z" (to-string (Timestamp. 893462400000))))
   (is (= "1998-04-25T00:00:00.000Z" (to-string "1998-04-25T00:00:00.000Z"))))
 
+(deftest test-to-edn
+  (is (nil? (to-edn nil)))
+  (is (nil? (to-edn "")))
+  (is (nil? (to-edn "x")))
+  (is (= (date-time 1970 1 1 0 0 0 0)
+         (edn/read-string {:readers data-readers}
+                          (to-edn 0))))
+  (are [o] (= (date-time 1998 4 25 0 0 0 0)
+               (edn/read-string {:readers data-readers}
+                                (to-edn o)))
+    (date-time 1998 4 25)
+    (date-midnight 1998 4 25)
+    (date-midnight 1998 4 25)
```

Comment:
Should this second `date-midnight` line be some other call?

Reply:
It is a paste error.  The tests are intended to be the same cases as in test-to-string.  Thank you for the catch.